RecoTracker/Configuration/python/customizeMinPtForHitRecoveryInGluedDet.py

```diff
@@ -0,0 +1,10 @@
+def esproducers_by_type(process, *types):
+    return (module for module in process._Process__esproducers.values() if module._TypedParameterizable__type in types)
+def customizeMinPtForHitRecoveryInGluedDet(process,value):
+   for esp in esproducers_by_type(process, "Chi2MeasurementEstimatorESProducer", "Chi2ChargeMeasurementEstimatorESProducer"):
+       esp.MinPtForHitRecoveryInGluedDet = value
```

Comment:
For what concern HLT, my understanding is that as soon this PR will be integrated a new config will be created with the new parameter made explicit.
@mtosi, @Martin-Grunewald, @fwyzard to confirm